backend/api/v1/debug_interceptor.go

```diff
@@ -31,29 +32,36 @@ func NewDebugInterceptor(errorRecordRing *api.ErrorRecordRing) *DebugInterceptor
 
 // DebugInterceptor is the unary interceptor for gRPC API.
 func (in *DebugInterceptor) DebugInterceptor(ctx context.Context, request any, serverInfo *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (any, error) {
+	startTime := time.Now()
 	resp, err := handler(ctx, request)
 	if err != nil {
-		in.debugInterceptorDo(ctx, request, serverInfo.FullMethod, err)
+		in.debugInterceptorDo(ctx, request, serverInfo.FullMethod, err, startTime)
 	}
 
 	return resp, err
 }
 
 // DebugStreamInterceptor is the unary interceptor for gRPC API.
 func (in *DebugInterceptor) DebugStreamInterceptor(request any, ss grpc.ServerStream, serverInfo *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
+	startTime := time.Now()
 	err := handler(request, ss)
 	ctx := ss.Context()
 	if err != nil {
-		in.debugInterceptorDo(ctx, request, serverInfo.FullMethod, err)
+		in.debugInterceptorDo(ctx, request, serverInfo.FullMethod, err, startTime)
 	}
 
 	return err
 }
 
-func (in *DebugInterceptor) debugInterceptorDo(ctx context.Context, request any, fullMethod string, err error) {
+func (in *DebugInterceptor) debugInterceptorDo(ctx context.Context, request any, fullMethod string, err error, startTime time.Time) {
 	st := status.Convert(err)
-	if st.Code() == codes.Internal {
-		slog.Error("Internal error intercepted", slog.String("method", fullMethod), log.BBError(err), slog.Any("request", request))
+	switch st.Code() {
```

Comment:
f := slog.Debug
switch st.Code() {
case codes.Unauthenticated, codes.OutOfRange, codes.PermissionDenied, codes.NotFound:
  // pass.
default:
  f = slog.Error
}
f(...)

Reply:
Or use

```
			slog.Log(ctx, level, )
```

Reply:
but the msg field is different between client error and server error

Reply:
Just use the same message field "" or "processed request". Or a similar way to set different messages.